inventory/components/header.tsx

```
@@ -0,0 +1,223 @@
+                <div className={styles['user-dropdown']} ref={dropdownRef}>
+                  <button
+                    className={styles['user-btn']}
+                    type="button"
+                    onClick={() => setDropdownOpen((open) => !open)}
+                  >
+                    <Avatar>
+                      <AvatarImage src={user.photoUrl} alt={user.displayName} />
+                      <AvatarFallback>{user.displayName[0]}</AvatarFallback>
+                    </Avatar>
+                    <span className={styles['user-name']}>{user.displayName}</span>
+                    <svg className={`ml-1 w-4 h-4 transition-transform ${dropdownOpen ? styles['rotate-180'] : styles['rotate-0']}`} fill="none" stroke="currentColor" strokeWidth="2" viewBox="0 0 24 24"><path strokeLinecap="round" strokeLinejoin="round" d="M19 9l-7 7-7-7" /></svg>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)